py/selenium/webdriver/common/service.py

```diff
@@ -69,11 +69,10 @@ def start(self):
         try:
             cmd = [self.path]
             cmd.extend(self.command_line_args())
-            self.process = subprocess.Popen(cmd, env=self.env,
-                                            close_fds=platform.system() != 'Windows',
-                                            stdout=self.log_file,
-                                            stderr=self.log_file,
-                                            stdin=PIPE)
+            if any("hide_console" in arg for arg in self.command_line_args()):
+                self.process = subprocess.Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=PIPE, creationflags=0x08000000)
+            else:
```

Comment:
* Passing `creationflags` to `subprocess.Popen` will break on all non-Windows platforms and raise a `ValueError`.
* the `env` and `close_fds` were removed
* args were changed to `stdin=PIPE, stdout=PIPE, stderr=PIPE`, which breaks log file redirection